compiler/runtime/RelocationTarget.hpp

```diff
@@ -0,0 +1,36 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2019 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+#ifndef TR_RELOCATION_TARGET_HPP
+#define TR_RELOCATION_TARGET_HPP
```

Comment:
Some of your include files you say `_HPP`, others you say `_INCL`.  Please be consistent.  I believe `_INCL` is more consistent with the rest of the codebase.

Reply:
There are no more _HPP define guards in my commits. There is an  _HPP comment that will be fixed and shipped with the next commit. There are some _HPP include guards in compiler folder though, shall it be addressed in a new beginner issue?